packages/golang/domains/adminroles.go

```diff
@@ -468,3 +475,27 @@ func UpdateAdminRoleByID(roleID string, permissions []*AdminRolePermission) *err
 	}
 	return UpdatePermissionsForRole(roleID, permissions)
 }
+func ValidateRoleAndPermissions(roleID string, permissions []*AdminRolePermission) *errors.VironError {
+	// カンマをvalidate()で除く理由
+	// casbinのpolicyに "," が存在すると、policyをloadする際に不具合が出る
+	if roleID != "" {
+		if existsCommas(roleID) {
+			return errors.Initialize(http.StatusInternalServerError, "role id cannot contain commas.")
```

Comment:
Using StatusBadRequest could be more descriptive than StatusInternalServerError.

Reply:
[fix](https://github.com/cam-inc/viron/pull/845/commits/65b3723eab8a1190a3ad60498ffe4f51ec2f8f66)